commit 0d4261ad5d0028b26cd88e645b4507eed8aab3f7
Author: Colleen Twitty <colleen@cozybit.com>
Date:   Wed May 8 11:46:00 2013 -0700

    mac80211: enable Auth Protocol Identifier on mesh config.
    
    Previously the mesh_auth_id was disabled.  Instead set the
    correct mesh authentication bit based on the mesh setup.
    
    Signed-off-by: Colleen Twitty <colleen@cozybit.com>
    Signed-off-by: Johannes Berg <johannes.berg@intel.com>

diff --git a/net/mac80211/mesh.c b/net/mac80211/mesh.c
index 6952760881c8..c13db9ad394b 100644
--- a/net/mac80211/mesh.c
+++ b/net/mac80211/mesh.c
@@ -748,7 +748,6 @@ int ieee80211_start_mesh(struct ieee80211_sub_if_data *sdata)
 	ieee80211_configure_filter(local);
 
 	ifmsh->mesh_cc_id = 0;	/* Disabled */
-	ifmsh->mesh_auth_id = 0;	/* Disabled */
 	/* register sync ops from extensible synchronization framework */
 	ifmsh->sync_ops = ieee80211_mesh_sync_ops_get(ifmsh->mesh_sp_id);
 	ifmsh->adjusting_tbtt = false;